### PR TITLE
Add @types/svgdom

### PR DESCRIPTION
**Pull request by altano**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/svgdom/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export function createSVGWindow(): SVGWindow;
+export function createSVGDocument(): SVGDocument;
+
+export function createWindow(): Window;
+export function createDocument(): Document;
+
+export function createHTMLWindow(): Window;
+export function createHTMLDocument(): Document;
+
+/*~ You can declare types that are available via importing the module */
```
Comment (peterblazejewicz) [inline types/svgdom/index.d.ts:17]:
```suggestion
```
this came from template inline guides

**Comment by peterblazejewicz** [comment]
do you need this change to align with modern module exports?
https://github.com/svgdotjs/svgdom/blob/master/package.json#L5C3-L9C5
```json
// package.json
{
    "private": true,
    "type": "module",
    "main": "./main-require.cjs",
    "exports": {
        "import": "./main-module.js",
        "require": "./main-require.cjs"
    }
}
```

**Comment by altano** [comment]
> do you need this change to align with modern module exports?

Are you referring to this bit from the readme?

> The second reason to create your own package.json is to specify ES modules. If the implementation package uses ESM and specifies "type": "module", then you should add a package.json with the same:

If so, yeah I should do that. I didn’t initially see it. Will post an update later.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@altano thanks!

**Comment by altano** [comment]
Ready to merge

**Comment by cyberwombat** [comment]
This is missing some definitions such as:   setFontDir,   setFontFamilyMappings,   preloadFonts, config... perhaps more

**Comment by altano** [comment]
> This is missing some definitions such as: setFontDir, setFontFamilyMappings, preloadFonts, config... perhaps more

Yep: please feel free to contribute to the types! I’m happy to review.

**Pull request merged**